yt/frontends/gizmo/tests/test_outputs.py

```diff
@@ -80,7 +80,7 @@ def test_gas_particle_fields():
     
     ptype = "gas"
     derived_fields = []
-    for species in ["H", "H_p0", "H_p1"]:
+    for species in ["H_p0", "H_p1"]:
```

Comment:
What is the rationale behind not testing "H" anymore?

Reply:
It doesn't exist anymore https://github.com/yt-project/yt/pull/2336